src/com/facebook/buck/jvm/kotlin/KotlinConfiguredCompilerFactory.java

```diff
@@ -73,4 +85,69 @@ public CompileToJarStepFactory configure(
   private Javac getJavac(BuildRuleResolver resolver, @Nullable JvmLibraryArg arg) {
     return javacFactory.create(new SourcePathRuleFinder(resolver), arg);
   }
+
+  private ImmutableList<String> condenseCompilerArguments(CoreArg kotlinArgs) {
```

Comment:
seems like there is no existing test for `KotlinConfiguredCompilerFactory.java`. you can probably add one for this method.

Reply:
any good prior examples you could point me to?

Reply:
checkout https://github.com/facebook/buck/blob/master/test/com/facebook/buck/jvm/kotlin/KotlinBuckConfigTest.java for unit tests example

for integration tests https://github.com/facebook/buck/blob/master/test/com/facebook/buck/jvm/kotlin/KotlinLibraryIntegrationTest.java

there are bunch of test data https://github.com/facebook/buck/tree/master/test/com/facebook/buck/jvm/kotlin/testdata, you can probably add an integration test too.